tools/cli_install.php

```diff
@@ -163,6 +163,10 @@
    $entityConfig->getFromDBByCrit([
       'entities_id' => '0',
    ]);
+   $entityConfig->update([
+      'id'           => $entityConfig->getID(),
+      'download_url' => PLUGIN_FLYVEMDM_AGENT_BETA_DOWNLOAD_URL
+   ]);
    $devquery = "UPDATE `glpi_plugin_flyvemdm_entityconfigs`
```

Comment:
Hi @btry 

Shouldn't these lines ($devquery) be removed from the code? 

Since the new code makes the update

Reply:
You're right. This was a draft when I commented this part of code, and I forgot to drop ip... I'll clean it up